gnosis/eth/clients/blockscout_client.py

```diff
@@ -48,9 +48,6 @@ class BlockscoutClient:
         EthereumNetwork.EVMOS_TESTNET: "https://evm.evmos.dev/graphiql",
         EthereumNetwork.KCC_MAINNET: "https://scan.kcc.io/graphiql",
         EthereumNetwork.KCC_TESTNET: "https://scan-testnet.kcc.network/graphiql",
-        EthereumNetwork.ARBITRUM_ONE: "https://explorer.arbitrum.io/graphiql",
-        EthereumNetwork.ARBITRUM_NOVA: "https://nova-explorer.arbitrum.io/graphiql",
-        EthereumNetwork.ARBITRUM_GOERLI: "https://goerli-rollup-explorer.arbitrum.io/graphiql",
```

Comment:
Why are you removing other blockscout chains different to Arbitrum Sepolia?

Reply:
These endpoints are not available and we (Offchain Labs) will not be bringing them back. The one blockscout instance still online(Nova) has the graphiql endpoint disabled. Do you want this in a different PR?

Reply:
No, it is ok then.